example-fileupload/README.md

```diff
@@ -0,0 +1,507 @@
+        console.log(response); // Success
+        this.allFiles = response.files // if success then set the allFiles variable to the files returned from server
+    }, function(error) {
+        console.error(error); // If we fail then print the error to the console.
+    });
+} 
+```
+This function does the following:
+
+ 1. Requests all the files from the server
+ 2. When the request is processed from the server if it's successful then set the allFiles variable to the file list returned
+ 3. If we fail then log the error into the console for further inspection and debugging.
+
+and finally, we will create a `created()` function, in VueJS the `created()` function is always executed when the component is created hence its name, we use this to trigger the initial get of data when the `App.vue` component is first created. Go ahead and add this code into the `export default` object and make sure it isn't in the `methods`object or else it will not execute
+```js
+created() {
+  this.getAllFiles() // Trigger initial get of file list.
+}
+```
+With that, all the code for the `App.vue` component is done!
+
+You should finish off with something like this:
+```js
+<script>
+import { appwrite } from './utils'
+
+export default {
+ name: 'App',
+ components: {
+ },
+ data() {
+   return {
+     allFiles: []
+   }
+ },
+ created() {
+   this.getAllFiles()
+ },
+ methods: {
+   getAllFiles: function() {
+     let promise = appwrite.storage.listFiles();
+
+      promise.then((response) => {
+          console.log(response); // Success
+          this.allFiles = response.files
+      }, function (error) {
+          console.error(error); // Failure
+      });
+   }
+ }
+}
+</script>
+```
+
+
+### Styling
+When it comes to styling this app feel free to style it yourself and make it unique or use the style I created earlier if you want to use mine simply replace the `<style>` section with the following code:
+```html
+<style>
+  * {
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  }
+
+  .FileCore {
+    width: 500px;
+    height: auto;
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: translateX(-50%) translateY(-50%);
+    box-shadow: 0px 0px 10px black;
+    padding: 30px;
+    border-radius: 10px;
+    text-align: center;
+    max-height: 80%;
+    overflow-y: auto;
+  }
+</style>
+```
+
+and finally, the `App.vue` is complete (until we create the other two components and need to add them into here.)
+
+Give yourself a pat on the back. 🥳🥳🥳
+
+Next, we will start creating the fileUploader.
+
+## `src/components/FileUploader.vue`
+If you haven't already, go ahead and create the `src/components/FileUploader.vue` file, Now before we start creating this component I'm going to start talking about how it's going to work.
+Essentially all the FileUploader appears as to the user will be a button but behind the scenes, we're going to pull some trickery to make it easy to start file uploads.
```

Comment:
This sentence needs to be rewritten in my opinion

Reply:
I'm not sure if it needs to be rewritten entirely, I agree it should be revised on but I think we should just clean it up and make it a bit clearer explaining what's going on with this component.

Reply:
That's fine too

Reply:
@PineappleIOnic how is this going?